egs/iam/s5/local/unk_arc_post_to_transcription.py

```diff
@@ -1,14 +1,35 @@
 #!/usr/bin/env python
 
+""" This module will be used by scripts for open vocabulary setup.
+ If the hypothesis transcription contains <unk>, then it will replace 
+ the <unk> with the word predicted by <unk> model. It is currently 
+ supported only for triphone setup.
+
+ Args:
+  phones: File name of a file that contains the phones.txt, (symbol-table for phones).
+          phone and phoneID, Eg. a 217, phoneID of 'a' is 217. 
+  words: File name of a file that contains the words.txt, (symbol-table for words). 
+         word and wordID. Eg. ACCOUNTANCY 234, wordID of 'ACCOUNTANCY' is 234.
+  unk: ID of <unk>. Eg. 231
+  input-ark: file containing hypothesis transcription with <unk>
+  out-ark: file containing hypothesis transcription without <unk>
```

Comment:
The input/output are not in ark format. Maybe rename input-ark to 1best-arc-post, and out-ark to output-text. The description of input-ark is misleading. It should be something like: A file in arc-post format, which is a list of timing info and posterior of arcs along the one-best path from the lattice. Also it's better to change the description of output-text to: File containing hypothesis transcription with \<unk\> recognized by the unk-model.